encord/project.py

```diff
@@ -209,7 +209,8 @@ def list_label_rows_v2(
         )
 
         label_rows = [
-            LabelRowV2(label_row_metadata, self._client, self._ontology) for label_row_metadata in label_row_metadatas
+            LabelRowV2(label_row_metadata, self._client, self._ontology)  # type: ignore
```

Comment:
I think mypy can be more explicit on what to ignore. I usually try to be most explicit